frontend/css/map.css

```diff
@@ -0,0 +1,105 @@
+body {
+    margin: 0 0 0 0;
+    padding: 0 0 0 0;
+    overflow-y: hidden;
+}
+/* HEADER BOX */
+header {
+    background-color: rgb(167, 60, 113);
+    filter: drop-shadow(0px 10px 10px rgba(0,0,0,.15));
+    width: 100vw;
+    height: 11vh;
+    position: absolute;
+    margin: 0 0 0 0;
+    z-index: 2;
+}
+#bus-name {
+    color: white;
+    margin: 0 0 0 25%;
+    padding: 0 0 0 0;
+    font-family: 'Lato';
+    font-size: 22px;
+    height: 11vh;
+    line-height: 11vh;
+    text-align: center;
+    width: 50vw;
+    filter: none;
+    position: absolute;
+    z-index: 3;
+}
+
+/* MAP CONTENT */
+#map {
+    position: absolute; 
+    top: 11vh;
+    bottom: 0; 
+    width: 100%;
+    height: 79vh;
+    z-index: 0;
+}
+
+.marker { 
+    width: 70px;
+    height: 18px;
+	font-size: 12px;
+}
+.usr {
+    color: rgb(240, 119, 198);
+    font-family: 'Lato';
+    font-size: 13px;
+    margin-top: 6px;
+    display: inline;
+}
+.time {
+    font-family: 'Lato';
+    font-size: 10px;
+    margin: 0 10px 0 15px;
+    color: rgba(0,0,0,0.7);
+    display: inline;
+    float: right;
+}
+.msg {
+    font-family: 'Lato';
+    font-size: 15px;
+}
+.mapboxgl-popup {
+    filter: drop-shadow(3px 0 0 rgb(223, 98, 179)) 
+        drop-shadow(0 3px 0 rgb(223, 98, 179))
+        drop-shadow(-3px 0 0 rgb(223, 98, 179)) 
+        drop-shadow(0 -3px 0 rgb(223, 98, 179));
+}
+.mapboxgl-popup-content {
+    border-radius: 10px;
+}
+.mapboxgl-popup-close-button {
+    margin-top: 3px;
+}
+
+
+
+/* FOOTER */
+footer {
+    background-color: white;
+    filter: drop-shadow(0px -10px 10px rgba(0,0,0,.15));
+    width: 100vw;
+    height: 10vh;
+    position: absolute;
+    bottom: 0;
+    z-index: 2;
+}
+footer div {
+    width: 100vw;
+    height: 10vh;
+    line-height: 10vh;
+    text-align: center;
+    display: flex;
+    justify-content: space-around;
+}
+.icon {
+    height: 5vh;
+    line-height: 10vh;
+}
+footer div a {
+    width: 50vw;
+    padding: 2.5% 0 0 0;
+}
```

Comment:
once we've gotten rid of the footer from our HTML we'll also be able tog remove this CSS (except maybe ur `.icon` class which still might be useful in style those links in the header)